Documentation/ExtensionArchitecture/Extbase/Reference/_FrontendPlugin/_ExternalRedirectController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Psr\Http\Message\ResponseInterface;
+use TYPO3\CMS\Extbase\Mvc\Controller\ActionController;
+
+// Example by: https://brot.krue.ml/extbase-controller-action-responses-in-typo3/
+final class MyController extends ActionController
+{
+    public function downloadAction(): ResponseInterface
```

Comment:
downloadAction might not be the best name for performing a redirect.

Reply:
Renamed this.